src/contracts/middleware/StakeRegistry.sol

```diff
@@ -92,6 +85,32 @@ contract StakeRegistry is StakeRegistryStorage {
         return _totalStakeHistory[quorumNumber][index];
     }
 
+    /// @notice Returns the indices of the operator stakes for the provided `quorumNumber` at the given `blockNumber`
+    function getStakeUpdateIndexForOperatorIdForQuorumAtBlockNumber(bytes32 operatorId, uint8 quorumNumber, uint32 blockNumber)
+        external
+        view
+        returns (uint32)
+    {
+        return _getStakeUpdateIndexForOperatorIdForQuorumAtBlockNumber(operatorId, quorumNumber, blockNumber);
+    }
+
+
+    /// @notice Returns the indices of the total stakes for the provided `quorumNumbers` at the given `blockNumber`
+    function getTotalStakeIndicesByQuorumNumbersAtBlockNumber(uint32 blockNumber, bytes calldata quourmNumbers) external view returns(uint32[] memory) {
```

Comment:
typo again on spelling of 'quorum'

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/81/commits/802ff35f8f19ae61cc7ae882d1d55f7327d59687